src/dags/exit_node.hpp

```diff
@@ -35,13 +35,15 @@ class ExitNode : public Node {
     ResponseType* response;
     const tensor_map_t outputsInfo;
     bool useSharedOutputContent;
+    const std::string pipelineName;
 
 public:
-    ExitNode(ResponseType* response, const tensor_map_t& outputsInfo, std::set<std::string> gatherFromNode = {}, bool useSharedOutputContent = true) :
+    ExitNode(ResponseType* response, const tensor_map_t& outputsInfo, std::set<std::string> gatherFromNode = {}, bool useSharedOutputContent = true, const std::string pipelineName = "") :
```

Comment:
With `const std::string pipelineName = ""` there is high chance of perf drop since we copy the name each time we create pipeline (each time new request is received)